.vscode/launch.json

```diff
@@ -0,0 +1,53 @@
+{
+    //################################################################################
```

Comment:
These copyright header inside json file doesn't make sense.
Can you confirm the impact on adding .vscode files to gitignore? as these are IDE specific configurations and should not be present on open source repository.